File: Data Structures & Algorithms/excel-sheet-column-title/submission-1.cpp
class Solution {
public:
    string convertToTitle(int n) {
        string ans = "";
        while(n>0){
            n--;
            int rem = n%26;

            ans = char('A'+rem)+ans;
            n = n/26;
        }
        return ans;
        
    }
};

File: Data Structures & Algorithms/simplify-path/submission-1.java
class Solution {
    public String simplifyPath(String path) {
        Stack<String>st = new Stack<>();
        String[] parts = path.split("/");

        for(String part:parts){
            if(part.equals("") || part.equals(".")){
                continue;
            }else if(part.equals("..")){
                if(!st.isEmpty()){
                    st.pop();
                }
            }else{
                st.push(part);
            }
        }
        if(st.isEmpty()){
            return "/";
        }
        StringBuilder ans = new StringBuilder();
        for(String s:st){
            ans.append("/").append(s);
        }
        return ans.toString();
        
    }
}